NBXplorer/Repository.cs

```diff
@@ -1149,67 +1173,154 @@ public Task<TrackedTransaction[]> GetMatches(NBitcoin.Transaction tx, uint256 bl
 		{
 			return GetMatches(new[] { tx }, blockId, now, useCache);
 		}
+
+		private bool isTaproot(TxIn input)
+		{
+			// return true;
+			return input.ScriptSig.IsScriptType(ScriptType.Taproot);
```

Comment:
The whole problem is that this is not possible. You can't know if a txin is taproot just by looking the input.ScriptSig.

For this reason, I think taproot input/outputs should not be treated differently from the rest.

Reply:
Okay will do that.

Reply:
Okay will do that.